gtk/rgdebinstallprogress.cc

```diff
@@ -293,14 +292,6 @@ void RGDebInstallProgress::startUpdate()
    // check if we run embedded
    int id = _config->FindI("Volatile::PlugProgressInto", -1);
    if (id > 0) {
-#if !GTK_CHECK_VERSION(3,0,0)
-      GtkWidget *vbox = GTK_WIDGET(gtk_builder_get_object(_builder, "vbox_rgdebinstall_progress"));
-      _sock =  gtk_plug_new(id);
-      gtk_widget_reparent(vbox, _sock);
-      gtk_widget_show(_sock);
-#else
```

Comment:
We need to keep the "else" part here to indicate that this is not supported. I guess we could also just drop this feature entirely, i.e. remove the "int id = " and the if part and just keep "show();". But best to do this as a separate PR to keep the changes that are very mechanical (like doing #if/#then) separate from the ones that actually change behavior. Does that make sense?